pandas/tests/reshape/test_concat.py

```diff
@@ -2350,6 +2350,21 @@ def test_concat_datetime_timezone(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_concat_series_name_npscalar_tuple(self):
```

Comment:
parameterize this on a np.int64 and an int

Reply:
done - took the test out of class, parameterised as suggested, also actually removed the test case which took a row from DataFrame